satpy/multiscene.py

```diff
@@ -490,8 +521,55 @@ def _simple_frame_compute(self, writers, frame_keys, frames_to_write):
                 w = writers[frame_key]
                 w.append_data(product_frame.compute())
 
+    def _get_writers_and_frames(
+            self, filename, datasets, fill_value, ignore_missing,
+            enh_args, imio_args):
+        """Get writers and frames.
+
+        Helper function for save_animation.
+        """
+        scene_gen = self._scene_gen
+        writers = {}
+        frames = {}
```

Comment:
these aren't needed before line 552 where they are reinitialised.

Reply:
You're right.  I thought `flake8` would pick that up (unused variable) but apparently not.  Fixed in 442e44a96c1ded069f6fc61869d5316b42a03c9f